unsupported/Eigen/src/SpecialFunctions/SpecialFunctionsImpl.h

```diff
@@ -1574,6 +1574,8 @@ struct betainc_impl<double> {
   }
 };
 
+#endif  // EIGEN_HAS_C99_MATH
```

Comment:
this `#endif` has been moved around a bit. is this intended?

Reply:
Yes.

The initial checkin broke the TF build, because for some reason when we compile non gpu code with TF, the "EIGEN_HAS_C99_MATH" macro does not get set (it should, but it does not) resulting in the TF build breaking when trying to compile the newly added `i0e` and `i1e` math functions in `Eigen`.

The same fix has also been applied to the official Eigen repo, it just has not shown up here yet (the git mirror seems to be almost a week out-of-sun with the official bitbucket repo....dont know why)

Reply:
got it